testing/regress/hpcc/regression/suite.py

```diff
@@ -126,8 +126,17 @@ def addFileToSuite(self, eclfile):
             versions = eclfile.getVersions()
             versionId = 1
             for version in versions:
-                files.append({'basename':basename, 'version':version,  'id':versionId })
-                versionId += 1
+                if 'no'+self.name in version:
+                    # Exclude it from this target
+                    pass
+                else:
+                    # Remove exclusion key(s) from version string
+                    config = getConfig();
```

Comment:
Adding nothor2 to the clusters did allow me to run it.

However //nothor should mean do no run this on the thor platform, not do not run this on the thor cluster.

There should not be any reference to cluster names in any of the files.